src/reference/informative/informC08_interpretation_of_mathematics2.rst

```diff
@@ -0,0 +1,174 @@
+.. _informC08_interpretation_of_mathematics2:
+
+.. container:: toggle
+
+  .. container:: header
+
+    See more
+
+  .. container:: infospec
+
+    .. container:: heading3
+
+      Understanding how the mathematics works
+
+    The mathematics of a mathematical model is a collection of statements which are held to be true.
```

Comment:
Maybe also mention initial values here, which are not held to be true at all times?

Reply:
Done :)